src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] It would be better to fix the prop-types warning instead of disabling it. Fixing it helps us write clean and readable code